Samples/Sources/SampleClusterTracing/Cooking/Chopping.swift

```diff
@@ -0,0 +1,44 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Distributed Actors open source project
+//
+// Copyright (c) 2018-2022 Apple Inc. and the Swift Distributed Actors project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.md for the list of Swift Distributed Actors project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import _PrettyLogHandler
+import Distributed
+import DistributedCluster
+import Logging
+import NIO
+import Tracing
+
+protocol Chopping {
+    func chop(_ vegetable: Vegetable) async throws -> Vegetable
+}
+
+distributed actor VegetableChopper: Chopping {
+    @ActorID.Metadata(\.receptionID)
+    var receptionID: String
+
+    init(actorSystem: ActorSystem) async {
+        self.actorSystem = actorSystem
+
+        self.receptionID = "*" // default key for "all of this type"
```

Comment:
Is this the only type that gets registered with the receptionist so it doesn't matter what `receptionID` is?

Reply:
`*` is by  convention the "default key" which we use for "all the actors" https://github.com/apple/swift-distributed-actors/issues/1084